fastchat/model/model_adapter.py

```diff
@@ -571,6 +571,32 @@ def match(self, model_path: str):
     def get_default_conv_template(self, model_path: str) -> Conversation:
         return get_conv_template("h2ogpt")
 
+class Robin7bAdapter(BaseAdapter):
+    """The model adapter for LMFlow/Full-Robin-7b-v2"""
+
+    def match(self,model_path:str):
+        return "Robin" in model_path
+
+    def get_default_conv_template(self,model_path:str) -> Conversation:
+        return get_conv_template("Robin")
+    
+class Robin13bAdapter(BaseAdapter):
+    """The model adapter for LMFlow/Full-Robin-13b-v2"""
+
+    def match(self,model_path:str):
+        return "Robin" in model_path
+
+    def get_default_conv_template(self,model_path:str) -> Conversation:
+        return get_conv_template("Robin")
+    
+class Robin33bAdapter(BaseAdapter):
```

Comment:
Merge the three adapters. We only need one.

Reply:
Hi. I have merged the three adapters